### ISSUE TITLE
推荐广告使用PaddleRec里的gnn算法，无法收敛

### ISSUE THREAD

**Issue by xugang02** [PaddlePaddle/Paddle#17539]
如果您没有查询到相似问题，为快速解决您的提问，建立issue时请提供如下细节信息：
- 标题：简洁、精准概括您的问题，例如“Insufficient Memory xxx" ”
- 版本、环境信息：
    1）PaddlePaddle版本：1.4.1
    2）CPU：
    3）GPU：CUDA9
    4）系统环境：python 2.7.3
- 训练信息
    1）单机，单
    2）Tesla K40m
    3）Operator信息
- 复现信息：
CUDA_VISIBLE_DEVICES=13,10,7,8,9,11,12,14 fluid -u train.py --use_cuda 1 >log1.txt 2>&1 &
- 问题描述：
使用PaddleRec里的gnn算法对用户对点击广告序列做预测，无法收敛比较好的结果，train_acc一直在0.6以下，麻烦帮忙看看
2019-05-21 17:58:31,923-INFO: epoch loss: 6.3719
2019-05-21 17:58:32,113-INFO: model saved in ./saved_model/epoch_13
2019-05-21 17:58:33,956-INFO: global_step: 141500, loss: 6.3611, train_acc: 0.4799
2019-05-21 17:59:00,168-INFO: global_step: 142000, loss: 6.4052, train_acc: 0.4773
2019-05-21 17:59:25,852-INFO: global_step: 142500, loss: 6.3732, train_acc: 0.4800
2019-05-21 17:59:52,234-INFO: global_step: 143000, loss: 6.3516, train_acc: 0.4793
2019-05-21 18:00:19,459-INFO: global_step: 143500, loss: 6.3832, train_acc: 0.4783
2019-05-21 18:00:46,142-INFO: global_step: 144000, loss: 6.3396, train_acc: 0.4831
2019-05-21 18:01:10,977-INFO: global_step: 144500, loss: 6.3737, train_acc: 0.4774
2019-05-21 18:01:35,072-INFO: global_step: 145000, loss: 6.3783, train_acc: 0.4773
2019-05-21 18:02:00,112-INFO: global_step: 145500, loss: 6.4053, train_acc: 0.4745

**Comment by hutuxian** [comment]
Suggestion: Hyperparameter tuning. Try different values of initial learning rate(lr), lr decay steps, lr decay rate and so on.
Feel free to reopen this issue if you have further questions.
